### PR TITLE
Info data change update

### PR DESCRIPTION
**Pull request by salichon**
Finalization of the Docuemtns about archive changes. 
Completed from the work from Elisabetta

### CODE REVIEW

File: data-changes/location-codes.md [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
-- derived products we distribute and were not able to be reprocessed might still refer to old location code. For example, quakeML might still show that a station had location code 20, but then miniseed data will have that station with a location code 21
+If you use strong motion miniseed data and metadata, you might be impacted by this change.
+Impacts are expected:
+- if you used and downloaded data between 2019 and 2021, you might not be able to retrieve the exact same dataset from the Geonet FDSN data service
+ as it has been queried at that time
+- if you have a local copy of our data, some strong motion stream naming will differ with our GeoNet archive
+- if you have a pre-October 2021 upload of Geonet station metadata into a database or a software querying dataset from 2011 to 2020
+- If you are querying GeoNet derived earthquake products: Some are not updated/reprocessed accordingly and will refer to old location code (namely `20`).
+For example, some events xml files (quakeML) or earthquake bulletins  might still have the forner strong motion station streams naming with location code 20,
```
Comment (mnaguit) [inline data-changes/location-codes.md:51]: from 'might still have the forner' to 'may still use the former'

File: data-changes/location-codes.md [hunk 2]
```diff
@@ -1,45 +1,55 @@
...
+- seismic derived data: picks ID and magnitude ID (distributed via FDSN) might have metadata not correctly associated
 
-If you use strong motion miniseed data and metadata, you might be impacted by this change. The impact we expect can be of two forms:
-- if you have used and downloaded data between 2019 and 2021, you might have not been able to retrieve the complete dataset from FDSN
-- if you have a local copy of our data, things might not match anymore with our GeoNet archive
-- derived products we distribute and were not able to be reprocessed might still refer to old location code. For example, quakeML might still show that a station had location code 20, but then miniseed data will have that station with a location code 21
+If you use strong motion miniseed data and metadata, you might be impacted by this change.
+Impacts are expected:
+- if you used and downloaded data between 2019 and 2021, you might not be able to retrieve the exact same dataset from the Geonet FDSN data service
+ as it has been queried at that time
```
Comment (mnaguit) [inline data-changes/location-codes.md:47]: shall we remove the phrase 'as it has been queried at that time'? I think the context is already captured even without this phrase

File: data-changes/location-codes.md [hunk 3]
```diff
@@ -1,45 +1,55 @@
...
+- if you have a local copy of our data, some strong motion stream naming will differ with our GeoNet archive
+- if you have a pre-October 2021 upload of Geonet station metadata into a database or a software querying dataset from 2011 to 2020
```
Comment (mnaguit) [inline data-changes/location-codes.md:49]: pre-October 2021?

File: data-changes/location-codes.md [hunk 4]
```diff
@@ -1,45 +1,55 @@
...
-We are now retroactively correcting our data archive and our metadata records, so that this change is properly represented.
+### Change description
+After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. 
+We found that a number of SM sensors were moved to a new location at the time without having their location code changed accordingly.
 
+We are now (2021) retroactively correcting our data archive and our metadata records, so that this change is properly represented.
 For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
 
-Due to the complexity of this work, there has been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+Due to the complexity of this work, there has been sometimes and  for a limited period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive.
```
Comment (mnaguit) [inline data-changes/location-codes.md:33]: Just a suggestion: 
replace with: "The complexity of this work is confined in the alignment of archived data with the metadata (e.g. changes in location codes).

File: data-changes/location-codes.md [hunk 5]
```diff
@@ -1,45 +1,55 @@
...
+### Change description
+After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. 
+We found that a number of SM sensors were moved to a new location at the time without having their location code changed accordingly.
 
+We are now (2021) retroactively correcting our data archive and our metadata records, so that this change is properly represented.
 For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
 
-Due to the complexity of this work, there has been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+Due to the complexity of this work, there has been sometimes and  for a limited period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. 
+This has now been fixed.
```
Comment (mnaguit) [inline data-changes/location-codes.md:34]: Suggestion:
This work may still be carried out in the future as the need arises, but we herein provided the major changes that have already been implemented.

File: data-changes/location-codes.md [hunk 6]
```diff
@@ -1,45 +1,55 @@
...
+## Location code definition
+Location ID: **A 2 character code used to uniquely identify different data streams at a single station.** These IDs are commonly used to logically separate multiple instruments or sensor sets at a single station.
 
 ## Background
-Location codes naming conventions used for GeoNet seismic stations are documented [on our _delta_ metadata repository](https://github.com/GeoNet/delta/tree/main/docs).
-Based on this nomenclature convention:
-- all strong motion sensors have a location code `2?`;
-- when a sensor is moved more than 1 m (but less than 200m), its location code will change accordingly (e.g. from `20` to `21`).
+- The Miniseedformat naming international conventions are available at (https://ds.iris.edu/ds/nodes/dmc/data/formats/seed/)  
+- The GeoNet seismic stations Location codes naming conventions are documented on "https://www.geonet.org.nz/data/supplementary/channels".
```
Comment (mnaguit) [inline data-changes/location-codes.md:16]: 'in' instead of 'on'

File: data-changes/location-codes.md [hunk 7]
```diff
@@ -1,45 +1,55 @@
...
-This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
-The change has been applied to data and metadata which have been corrected during 2021. 
+This document provides details of **Data/Metadata Legacy corrections** we have applied to correctly label the **location code** of **strong motion instruments** part of the GeoNet national network for periods of time between **2011 and 2020**.
+The change was applied to **data and metadata** from **27 seismic stations**  (in between June and October 2021) 
 
 You might be impacted by this change if you:
-- have a local copy of our data and/or metadata that is not up to date;
-- are using our data and metadata services while we apply the change;
-- are using a derived data product that has a mismatch in the metadata 
+- have a local copy of our data and/or metadata that is not up to date Prior to June/Oct 2021 ;
```
Comment (mnaguit) [inline data-changes/location-codes.md:7]: 'Prior' to 'prior'

File: data-changes/location-codes.md [hunk 8]
```diff
@@ -1,45 +1,55 @@
 # Location codes change
 
-This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
-The change has been applied to data and metadata which have been corrected during 2021. 
+This document provides details of **Data/Metadata Legacy corrections** we have applied to correctly label the **location code** of **strong motion instruments** part of the GeoNet national network for periods of time between **2011 and 2020**.
+The change was applied to **data and metadata** from **27 seismic stations**  (in between June and October 2021)
```
Comment (mnaguit) [inline data-changes/location-codes.md:4]: Is this what you mean?

"The change was applied between June & October 2021, to both data and metadata from 27 seismic stations".

File: data-changes/location-codes.md [hunk 9]
```diff
@@ -1,45 +1,55 @@
 # Location codes change
 
-This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
-The change has been applied to data and metadata which have been corrected during 2021. 
+This document provides details of **Data/Metadata Legacy corrections** we have applied to correctly label the **location code** of **strong motion instruments** part of the GeoNet national network for periods of time between **2011 and 2020**.
```
Comment (mnaguit) [inline data-changes/location-codes.md:3]: Suggestion:

"This document provides details of **Data/Metadata Alignment** that has been applied to correct the **location code** labels for a number of **strong motion stations** in the GeoNet national network. The correction in location codes primarily covers the time period betwwen **2011 to 2020** ". 

--note: we didn't really correct the data per se, we changed the filename of the data to align with the metadata

**Review by mnaguit — CHANGES_REQUESTED** [review]
Hi @salichon 
Thanks for this information update. 
Here are some of my suggestions, for your consideration. 
Cheers!

File: data-changes/location-codes.md (hunk 3 above)
Comment (salichon) [reply data-changes/location-codes.md]: yup - that s basically when most delta metadata were done  (20 to 21 )

Comment (mnaguit) [reply data-changes/location-codes.md]: oki

**Comment by salichon** [comment]
@elidana it is okay to me after the pass from you and @mnaguit 
thanks !

File: data-changes/location-codes.md (hunk 2 above)
Comment (salichon) [reply data-changes/location-codes.md]: changed by the betta 
all god

**Comment by elidana** [comment]
@salichon, do you want me to add Muriel's comment or can you do it?

File: data-changes/location-codes.md (hunk 3 above)
Comment (salichon) [reply data-changes/location-codes.md]: if you agree  (Pre october is when all the metadata were actually updated :))

**Comment by mnaguit** [comment]
Guys @elidana @salichon 
Feel free to ignore my suggestions...all good, no worries at all.
Just doing my job as a reviewer.

**Comment by salichon** [comment]
@elidana me doing it - :)

**Comment by elidana** [comment]
me say BIG THANKS!

File: data-changes/location-codes.md (hunk 5 above)
Comment (salichon) [reply data-changes/location-codes.md]: Added: "This type of archive change will still be carried in the future as the need arises althougyh to much smaller extend.
"

File: data-changes/location-codes.md (hunk 7 above)
Comment (salichon) [reply data-changes/location-codes.md]: reshaped by the betta

File: data-changes/location-codes.md (hunk 8 above)
Comment (salichon) [reply data-changes/location-codes.md]: change applied by the Betta :)

File: data-changes/location-codes.md (hunk 9 above)
Comment (salichon) [reply data-changes/location-codes.md]: added : "NB: This type of change in the archive only affects stream names: file headers only are modified
"

**Comment by salichon** [comment]
Last pass :) i  hope
cheers
jerome

**Review by elidana — APPROVED** [review]
looks good to me, thanks @salichon !

Can you please squash all commits before merging?

**Comment by salichon** [comment]
Yup @elidana after @mnaguit  last word :) 
I ll squash all this with great joy!

**Comment by salichon** [comment]
@mnaguit - all good with last changes ? 
thanks :)

**Comment by mnaguit** [comment]
Many thanks for your consideration @salichon

**Review by mnaguit — APPROVED** [review]

**Pull request merged**